examples/gallery/histograms/blockm.py

```diff
@@ -0,0 +1,63 @@
+"""
+Blockmean
+---------
+The :meth:`pygmt.blockmean` method calculates different quantities
+inside blocks/bins whose dimensions are defined via the ``spacing`` parameter.
```

Comment:
What defines a block vs. a bin? In the next sentence "block" is used for the average of the values while "bin" is used for the number of values. As far as I can tell, they represent the same thing (a two-dimensional area) in this example.

I'm not familiar with this kind of data analysis, but the term should be consistent if they mean the same thing.

Reply:
Yes it's the same, will change it :wink:, thanks @willschlitzer.

Reply:
I'll make a code suggestion